src/EthernetConnectionManager.cpp

```diff
@@ -128,13 +128,21 @@ void EthConnectionManager::check() {
           debugMessage(DebugLevel::Info, "Connected");
         }
         break;
+      case NetworkConnectionState::DISCONNECTING: {
```

Comment:
we added these ones back in?
I know they do nothing but I have always like them

Reply:
When there is a switch statement evaluating a enum the compiler can determine wether or not all possible value cases are covered within the switch statement (`-Wswitch-enum`). If one or more of those are not found the compiler issues a warning.

Usually missing cases indicate a programming problem because usually when one implements a switch-case with a enumeration type it is for implementing *finite state machines* (which is actually kind of what we do in the code). I've therefore added the empty if statements as a reminder for the programmer that there are states which are not covered and might present a possible problem (either in the future or already now).